### PR TITLE
Add bandit pre commit

### PR DESCRIPTION
**Pull request by kairoaraujo**
To achieve the OpenSSF Best Practices badge, we need to implement
the static code analysis tool.

This commit adds the `bandit` to the RSTUF CLI.

### CODE REVIEW

**Review by joshuagl — APPROVED** [review]
🎉

**Review by joshuagl — APPROVED** [review]

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,77 +1,83 @@
 -i https://pypi.org/simple
-alabaster==0.7.13; python_version >= '3.6'
-attrs==22.2.0; python_version >= '3.6'
-babel==2.11.0; python_version >= '3.6'
+alabaster==0.7.13 ; python_version >= '3.6'
```
Comment (MVrachev) [inline requirements-dev.txt:2]: Were the changes in this file required by `bandit`?

**Review by MVrachev — CHANGES_REQUESTED** [review]
Can you please look at the reason for the failing CI run?

File: requirements-dev.txt (hunk 1 above)
Comment (kairoaraujo) [reply requirements-dev.txt:2]: yes, it was added by the bandit.

**Comment by kairoaraujo** [comment]
@MVrachev  This is also ready for re-review

**Review by MVrachev — APPROVED** [review]

**Pull request merged**